changelogs/fragments/windows-deprecation-removal.yaml

```diff
@@ -0,0 +1,6 @@
+deprecated_features:
+- win_chocolatey - removed deprecated upgrade option and choco_* output return values
```

Comment:
As this gets converted into markdown I think you may need

```
- `win_chocolatey` - removed...
```

Reply:
Hmm, will see what @nitzmahone says, I haven't been doing them for my previous fragment entries but can start doing it now. Will also add to the porting guide as well.

Reply:
oh, it's actually generated as RST
https://raw.githubusercontent.com/ansible/ansible/stable-2.5/changelogs/CHANGELOG-v2.5.rst
I can see `win\_audit\_rule` as well as

```
The `stat`` and ``win_stat`` modules have changed
```

So maybe ignore the backticks for the moment, and we can check with @nitzmahone later?

Reply:
I haven't been too careful about formatting; for module bugfixes, I've been doing just `module_name - description of thing` like what's already here... We're still kinda playing around with what kinds of post-processing we want to do on these too, so this is fine with me for now. We can always go back and reformat consistently later if we come up with some standards.

Reply:
Have kept the formatting the same but added entries to the porting guide.